packages/zcli-apps/src/commands/apps/server.ts

```diff
@@ -37,10 +37,10 @@ export default class Server extends Command {
   static strict = false
 
   async run () {
-    const { flags } = this.parse(Server)
+    const { flags } = await this.parse(Server)
```

Comment:
Any reason for the extra `await`s? Not a critique, just out of curiosity

Reply:
Previous:
<img width="782" alt="image" src="https://user-images.githubusercontent.com/4507647/179910972-0b985e49-bf19-4eca-989c-39515513ffc5.png">

Now it's a promise:
<img width="803" alt="image" src="https://user-images.githubusercontent.com/4507647/179911171-34b03f8f-d81d-4d34-8c35-50e685c22768.png">